bootstrap.ps1

```diff
@@ -312,13 +312,6 @@ Function Install-West {
 }
 
 Function Main {

```

Comment:
Why where the parameters removed? Can PS5 not handle parameters?

Reply:
It can, but the alias for Main in the test files does not work.

The Main function is just a workaround to have the scripts tested without execution of the "main body". And I find the redefinition of the global parameters for the Main function rather useless.